README.md

```diff
@@ -6,11 +6,117 @@
 
 ## Introduction
 
-Adds the Pipeline `build` step, which triggers builds of other jobs.
+This plugin helps to add the pipeline `build` step, which triggers builds of other jobs.
 Use the [_"Pipeline Syntax" Snippet Generator_](https://jenkins.io/redirect/pipeline-snippet-generator) to get a detailed example for your build step.
 The Pipeline Syntax Snippet Generator helps the user generate steps for Jenkins pipeline.
 
-### Passing secret values to other jobs
+## Features
+
+The following features are available in Pipeline:
+
+### Project to Build
+This is the name or path of a downstream job to build. It may be another Pipeline job, but more commonly a freestyle or other project. Use a simple name if the job is in the same folder as this upstream Pipeline job; otherwise use relative paths like `../sister-folder/downstream` or absolute paths like `/top-level-folder/nested-folder/downstream`. Example;
+```groovy
+  job: 'Hello-World'
+```
+
+### Wait for completion
+You may ask that this Pipeline build wait for completion of the downstream build. In this case, the return value of the step is an object on which you can obtain read-only properties; so you can inspect its properties such as result using `.result` and so on.
+Properties of build upon completion are;
```

Comment:
Giving a few examples of how to use commonly needed properties like `number` or `result` in the context of the `build` step is great, but it is not really maintainable to enumerate all of these here. They are implemented in another plugin, to begin with, and code and docs would soon fall out of synch.

Better to point to mechanical docs. https://www.jenkins.io/doc/pipeline/steps/ does not seem to have it, but live inside Jenkins you can go to e.g. https://ci.jenkins.io/pipeline-syntax/globals#currentBuild